cfgov/wagtailadmin_overrides/views.py

```diff
@@ -0,0 +1,42 @@
+from wagtail.snippets.views.snippets import SnippetViewSetGroup
+
+from ask_cfpb.views import GlossaryTermViewSet
+from v1.views.snippets import (
+    ContactViewSet,
+    EmailSignUpViewSet,
+    PortalCategoryViewSet,
+    PortalTopicViewSet,
+    RelatedResourceViewSet,
+    ResourceViewSet,
+    ReusableTextViewSet,
+)
+
+
+class SnippetsViewSetGroup(SnippetViewSetGroup):
+    """Menu item that overrides the default Wagtail "Snippets" menu.
+
+    The default Wagtail "Snippets" menu includes all snippet models even if
+    those models have some other menu item configured for them. We'd prefer
+    if the default snippets menu only included those snippets that aren't
+    already exposed elsewhere.
+
+    See https://github.com/wagtail/wagtail/issues/11340 for more detail.
+
+    This item is implemented here in the wagtailadmin_overrides app instead
+    of in the v1 app because it relies on models from both the v1 and ask_cfpb
+    apps. Implementing it in v1 would introduce a circular dependency between
+    those apps.
```

Comment:
@willbarton any thoughts on this? I'm wary of adding any "permanent" code to the wagtailadmin_overrides app but this seemed like a minor improvement over [the current setup](https://github.com/cfpb/consumerfinance.gov/blob/57e544d885551cd0a95f5ee6f042e29d4180a30a/cfgov/v1/wagtail_hooks.py#L499-L512) where we import a model from ask_cfpb to register it in v1.wagtail_hooks. Having a third place to import both seemed like a slight improvement to the dependency graph, but I'm also happy to implement the current way if you think that's better.

Reply:
No, I think this makes as much sense as anywhere.

I don't love `wagtailadmin_overrides` becoming the CMS glue-package, but... absent any kind of broader reorganization, I do think this makes sense.

Any avoiding introducing new circular dependencies seems paramount to me.